packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -71,6 +71,7 @@ export type CallAdapterClientState = {
   devices: DeviceManagerState;
   endedCall?: CallState;
   isTeamsCall: boolean;
+  isPSTNorACSCall?: boolean;
```

Comment:
Maybe we can set it as callType - so it could be 'PSTN', 'ACS' or more, so we can add more call types for future compatibilities

Reply:
Is a growing list a breaking change? e.g. if the callType in a new version added 'pizzaParty' then contoso would hit a breaking change as they need to account for a new callType